Makefile

```diff
@@ -52,6 +52,13 @@ ifeq ($(LIBOPENSSL),0)
 	override LIB_DEPENDS += openssl
 endif
 
+GLIBC_MAJOR_VERSION := $(shell ldd --version | awk '/ldd/{print $NF}' | cut -f1 -d.)
+GLIBC_MINOR_VERSION := $(shell ldd --version | awk '/ldd/{print $NF}' | cut -f2 -d.)
+GLIBC_GE_2_25 := $(shell [ $(GLIBC_MAJOR_VERSION) -gt 2 -o \( $(GLIBC_MAJOR_VERSION) -eq 2 -a $(GLIBC_MINOR_VERSION) -ge 25 \) ] && echo true)
+ifeq ($(RH_GT_5_3),true)
+	CFLAGS += -DGLIBC_GE_2_25
```

Comment:
Whoa. Seems pretty RH specific, no?
And as such we probably don't need a 'glibc' define, but rather an 'HAVE_SYS_RANDOM'.
Can you use that instead of 'GLIBC_GE_2_25'?

Reply:
oh... my mistake, Thanks for checking, It didnt saved well